lib/BaselineAnalyzer.php

```diff
@@ -24,6 +24,7 @@ public function analyze(): AnalyzerResult
          * @var BaselineError $baselineError
          */
         foreach ($this->baseline->getIgnoreErrors() as $baselineError) {
+            $result->overallErrors += 1;
```

Comment:
```suggestion
            $result->overallErrors += $baselineError->count;
```

shouldn't we take the number from the error messages?

Reply:
actually this makes a lot of sense ;-).

fix incoming, good catch. thanks